README.md

```diff
@@ -21,6 +21,34 @@ There are two possible paths, depending on your choice of Kubernetes:
 1. For any Kubernetes, follow the [installation instructions][install].
 1. On MicroK8s, you simply have to enable the [Kubeflow add-on][addon].
 
+## Testing
+
+To deploy this bundle and run tests locally, do the following:
+1. Set up Kubernetes, Juju, and deploy the bundle you're interested in (`kubeflow` or `kubeflow-lite`) using the [installation guide](https://charmed-kubeflow.io/docs/install/)
+2. Install test prerequisites:
+```bash
+   sudo snap install juju-wait --classic
+   sudo apt update
+   sudo pip3 install -r requirements.txt
```

Comment:
Technically we are also missing installation instructions, for `pip3` although that could be skipped.

Charmcraft seems to be also needed in the tests.